src/Dapr.Client/DaprSubscribeConfigurationSource.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Grpc.Core;
+using Autogenerated = Dapr.Client.Autogen.Grpc.v1;
+
+namespace Dapr.Client
+{
+    internal class DaprSubscribeConfigurationSource : ConfigurationSource
+    {
+        private AsyncServerStreamingCall<Autogenerated.SubscribeConfigurationResponse> call;
+        private string id = string.Empty;
+
+        /// <summary>
+        /// Constructor.
+        /// </summary>
+        /// <param name="sourceStreamRequest">The request that is used to start the configuration streaming with Dapr.</param>
+        /// <param name="internalClient">The internal client used to call Dapr and begin a gRPC stream.</param>
+        /// <param name="callOptions">The options for the call to Dapr.</param>
+        internal DaprSubscribeConfigurationSource(
+            Autogenerated.Dapr.DaprClient internalClient,
+            Autogenerated.SubscribeConfigurationRequest sourceStreamRequest,
+            CallOptions callOptions) : base()
+        {
+            this.call = internalClient.SubscribeConfigurationAlpha1(sourceStreamRequest, callOptions);
```

Comment:
I think it makes more sense to put this code outside of this class. Eg: the source should be constructed with the `AsyncServerStreamingCall<>` rather than the client.